mobile/src/navigation/auth/common/constants.ts

```diff
@@ -0,0 +1,22 @@
+import { AppColor, AppFontFamily } from '~/common/enums/enums';
+
+const SCREEN_OPTIONS = {
+  tabBarLabelStyle: {
+    fontFamily: AppFontFamily.INTER_600,
+    fontSize: 14,
+    lineHeight: 16,
+    letterSpacing: 0.5,
```

Comment:
Try text-transform: 'none' to avoid uppercase

Reply:
I've tried it. That works, but produces an error, which doesn't crush the app
![error](https://user-images.githubusercontent.com/89215185/184115636-c6dcee18-9600-48fb-9f24-4b33cf4a6f07.png)
.

Reply:
```
import { MaterialTopTabNavigationOptions } from '@react-navigation/material-top-tabs';

const SCREEN_OPTIONS: MaterialTopTabNavigationOptions = 
```

Reply:
Thanks!